src/styles/theme.ts

```diff
@@ -0,0 +1,198 @@
+const white = '#FFF'
+const blue = '#218DFF'
+const blueDark = '#0b66c6'
+const bgLight = '#edf2f7'
+const bgDark = '#2e2e2e'
+const blackLight = '#181a1b'
+
+export enum ThemeType {
+  MAIN = 'main',
+  SECONDARY = 'secondary',
+}
+
+type SubTheme = {
+  [key in ThemeType]: {
+    [prop: string]: string
+  }
+}
+
+interface ColourThemeBase {
+  [t: string]: SubTheme | string | undefined
+}
+
+export interface ColourTheme extends ColourThemeBase {
+  bg1: string
+  button: SubTheme
+}
+
+const darkTheme: ColourTheme = {
+  bg1: bgDark,
+  bg2: blackLight,
+  button: {
+    main: {
+      text1: blue,
+      text1Hover: white,
+      get bg1(): string {
+        return darkTheme.bg1
+      },
+      bg1Hover: blueDark,
+    },
+    secondary: {
+      text1: blueDark,
+      text1Hover: white,
+      bg1: bgDark,
+      bg1Hover: blueDark,
+    },
+  },
+}
+
+const lightTheme: ColourTheme = {
+  bg1: bgLight,
+  bg2: white,
+  button: {
+    main: {
+      text1: white,
+      text1Hover: white,
+      bg1: blue,
+      bg1Hover: blueDark,
+    },
+    secondary: {
+      text1: blue,
+      text1Hover: white,
+      bg1: bgLight,
+      bg1Hover: blue,
+    },
+  },
+}
+
+export const colors = (darkMode: boolean): ColourTheme => {
+  const theme = darkMode ? darkTheme : lightTheme
+
+  return theme
+}
+
+// FOR REFERENCE
```

Comment:
ok, but this is from v1?

not sure if we will need it even as a reference. We have the colors in the original design. 
what do you think @biocom

Reply:
For the trading interface, I currently have these colors/variables defined here https://github.com/gnosis/dex-react/blob/0f9feb35788877e80aebbfe3027a8b56f3afbe83/src/components/layout/GenericLayout/variablesCss.ts

Reply:
Not sure if we want to consolidate theming between Simple Swap <-> Trading Interface. But I'd say in terms of colors they likely 90% overlap.

Reply:
cool @biocom this is what i needed. will transition my latest pr #1626